src/dispatch/static/dispatch/src/case/TimelineTab.vue

```diff
@@ -80,3 +77,17 @@ export default {
   },
 }
 </script>
+
+<style scoped>
+.wavy-underline {
+  text-decoration: underline;
+  text-decoration-style: wavy;
+  text-decoration-color: lightgray;
+  text-decoration-thickness: 1px;
+  text-underline-offset: 3px;
+}
+
+.pre-formatted {
+  white-space: pre;
+}
+</style>
```

Comment:
I wonder if we could move this to a styles file we can import from so we don't have the same code in two places.